scenarioo-server/src/main/java/org/scenarioo/business/diffViewer/ComparisonExecutor.java

```diff
@@ -336,7 +336,8 @@ private void logDuration(long startTime) {
 		String resolvedBaseBranchName = aliasResolver.resolveBranchAlias(baseBranchName);
 		for (ComparisonConfiguration comparisonConfiguration : comparisonConfigurations) {
 			String resolvedComparisonBranchName = aliasResolver.resolveBranchAlias(comparisonConfiguration.getBaseBranchName());
-			if (resolvedBaseBranchName.equals(resolvedComparisonBranchName)) {
+			if (resolvedBaseBranchName.equals(resolvedComparisonBranchName)
+				|| resolvedBaseBranchName.matches(resolvedComparisonBranchName)) {
```

Comment:
Please write a unit test for this. That will probably reveal a bug.

I think what you wanted to write was `resolvedComparisonBranchName.matches(resolvedBaseBranchName)`.

https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#matches(java.lang.String)

Reply:
@adiherzog the `reseolvedComparisonBranchName` is the one from the configuration, which is the regexp, which needs to go into the parameter, so I think it is correct and it worked when I tested it.